frontend/src/app/features/team-planner/team-planner/sidemenu/team-planner-sidemenu.component.ts

```diff
@@ -26,11 +28,15 @@ export class TeamPlannerSidemenuComponent extends UntilDestroyedMixin {
 
   @Input() projectId:string|undefined;
 
-  canAddTeamPlanner$ = this.currentUserService.hasCapabilities$(
-    'team_planners/create',
-    this.currentProjectService.id || undefined,
-  )
-    .pipe(this.untilDestroyed());
+  canAddTeamPlanner$ = this
+    .currentUserService
+    .hasCapabilities$(
+      'team_planners/create',
+      this.currentProjectService.id || undefined,
+    )
+    .pipe(
+      map((val) => val && !this.bannersService.eeShowBanners),
```

Comment:
in a sense the user does not have this capability and I'd really expect the API not to return the capability

Reply:
I agree , but it's easier to fix it pragmatically this way. I already added a technical debt for it though: https://community.openproject.org/projects/openproject/work_packages/41937/activity?query_id=862